Updated plot descriptsion

brianlangseth-NOAA opened the pull request:
This serves two major purposes:
1. Cleans up file naming structure where previously Rebuild_plot.R and do_project_plots.R were duplicative, and the function within do_project_plots didn't match the file name. 
2. Adds explanations for the plots should anyone decide to use them in the future and be at a loss of what they actually show. These are made as comments. I also made _slight_ changes to the plot themselves, which mostly serve to reproduce the plots from SUMMARY.xls

brianlangseth-NOAA committed:
Rename plot file #2

*Change most up to date plotting file (RebuildPlots) moved from r4ss to name of the function (DoProjectPlots)
*Delete old plot file

brianlangseth-NOAA committed:
Add comments for understanding figures

brianlangseth-NOAA committed:
Update the recovery plot

*Replace Tmax with vertical dashed line
*Add cumulative probability plots for reaching target (red) and currently at target (blue)

brianlangseth-NOAA committed:
Merge branch 'main' into updatePlots

brianlangseth-NOAA committed:
fix errors

*Fix recovery plot plotting and scale
*Add explanation for B0 histogram issue #5

brianlangseth-NOAA committed:
Add legend to recovery plot

brianlangseth-NOAA commented:
@chantelwetzel-noaa Can you glance over this? Mostly to ensure agreement on cleaning up the two files into one, and changing the name of do_project_plots.R to match the function's name (DoProjectPlots)

iantaylor-NOAA commented:
Thanks for continuing to improve this package @brianlangseth-NOAA!

You probably need to run `devtools::document()` to update the package documentation.
We could also set up a GitHub Action that automatically does that using `ghactions4r::use_update_roxygen_docs()`. See https://nmfs-fish-tools.github.io/ghactions4r/ for more detail.

brianlangseth-NOAA committed:
updated documentation

*run devtools::document() and update documentation
*required change a few variables from templates to params

iantaylor-NOAA committed:
add @brianlangseth-NOAA as additional package author
